client/src/components/App.jsx

```diff
@@ -24,11 +39,90 @@ class App extends React.Component {
       });
   }
 
+  /* componentWillUnmount() {
+    document.removeEventListener('mousedown', this.handleClickOutside);
+  } */
+
+  setWrapperRef(node) {
+    console.log('inside wrapper');
+    this.wrapperRef = node;
+    console.log(node);
+  }
+
+  handleOnChange(e) {
+    const stateObj = this.state;
+    if (stateObj.isVisible) {
+      this.setState({ isVisible: true });
+    }
+
+    const userId = 1;
+    const categoryId = e.target.value;
+    fetch(`filter/${userId}/${categoryId}`)
+      .then((response) => response.json())
+      .then((data) => {
+        this.setState({
+          username: data.user.username,
+          videos: data.videos,
+          currentCategory: data.videos[0].category.name,
+        });
+      });
+  }
+
+  handleOnRightClick() {
+    document.getElementById('container').scrollLeft += 50;
```

Comment:
Probably overkill, but since you scroll left and right by the same amount, consider making a 'const scrollDistance = 50' (around line 9???)